docs/zz_generated.seed.ee.yaml

```diff
@@ -79,7 +79,8 @@ spec:
           # found at https://azure.microsoft.com/en-us/global-infrastructure/locations/
           location: ""
         # Baremetal contains settings for baremetal clusters in datacenters.
-        baremetal: {}
+        baremetal:
```

Comment:
In https://github.com/kubermatic/kubermatic/tree/main/codegen/example-yaml please set an empty struct for tinkerbell, so the generator does not just output "null" for it.